hardware/raspberrypi/booteeprom.md

```diff
@@ -21,21 +21,25 @@ To check that the bootloader is working correctly, turn off the power, unplug ev
 
 If the Raspberry Pi is not booting it's possible that the bootloader EEPROM is corrupted. This can easily be reprogrammed using the Raspberry Pi Imager tool which is available via the [raspberrypi.org downloads page](https://www.raspberrypi.org/downloads/).
 
+Using the recovery image will erase any custom configuration options, resetting the bootloader back to factory defaults.
+
 ## Updating the bootloader
 
-We recommend setting up your Pi so that it automatically updates the bootloader: this means you will get new features and bug fixes as they are released. Bootloader updates are performed by the `rpi-eeprom` package, which installs a service that runs at boot-time to check for critical updates.
+Bootloader updates are instigated during a normal 'apt update`, `apt full-upgrade' cycle, this means you will get new features and bug fixes during your normal udpates. 
+
+Bootloader updates are performed by the `rpi-eeprom` package, which installs a service that runs at boot-time to check for critical updates. Although `rpi-eeprom` is available in the latest Raspberry Pi OS releases, there is usully no need to run it manually.
+
+To update your system, including the bootloader:
 
 ```
 sudo apt update
 sudo apt full-upgrade
-sudo apt install rpi-eeprom
 ```
 
 If you wish to control when the updates are applied you can disable the systemd service from running automatically and run `rpi-eeprom-update` manually.
 
 ```
-# Prevent the service from running, this can be run before the
-# package is installed to prevent it ever running automatically.
+# Disable
```

Comment:
This should probably be changed to "# Disable the systemd service" to make it clearer that it's only disabling the service, and not disabling `rpi-eeprom-update` itself?

Reply:
I think that's clear enough already